shared/constants/chat.js

```diff
@@ -626,7 +626,7 @@ const getSelectedInbox = (state: TypedState) => {
 const getTLF = createSelector(
   [getSelectedInbox, getSelectedConversation],
   (selectedInbox, selected) => {
-    if (isPendingConversationIDKey(selected)) {
+    if (selected && isPendingConversationIDKey(selected)) {
```

Comment:
this is in a different pr but i needed it also, just waiting on CI in that other pr